build/mvn

```diff
@@ -69,7 +69,7 @@ install_app() {
 
 # Install maven under the build/ folder
 install_mvn() {
-  local MVN_VERSION="3.3.3"
+  local MVN_VERSION="3.3.9"
```

Comment:
It's already in the enforcer rule, but oops, I did not update the expected Maven version in the parent POM. Coming now ...